CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.24
+
+<!-- generated comparing v5.0.0-alpha.23..next -->
+
+_Jan 25, 2021_
+
+Big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:
+
+- Converted 27 components to emotion
+- Fixed various issues on API documentation pages (#24526, #24503, #24504. #24517, #24417)
```

Comment:
````suggestion
- 📚 Fix various display issues on API documentation pages (#24526, #24503, #24504. #24517, #24417) @eps1lon.
- 📐 Add a subset of the system as flatten props on the CSS utility components (`Grid` and `Box` so far, `Typography` and `Stack` coming later) (#24485, #24499) @mnajdova.

  ```jsx
  <Box m={2}>
  ```
````

Reply:
One of these is from Marija. 

````suggestion
- 📚 Fix various display issues on API documentation pages (#24526, #24503, #24504. #24517, #24417).
- 📐 Add a subset of the system as flatten props on the CSS utility components (`Grid` and `Box` so far, `Typography` and `Stack` coming later) (#24485, #24499) @mnajdova.

  ```jsx
  <Box m={2}>
  ```
````